fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -673,6 +673,17 @@ private void resetAnalyzerAndStmt() {
         analyzer = new Analyzer(context.getCatalog(), context);
 
         parsedStmt.reset();
+
+        if (parsedStmt instanceof SelectStmt) {
```

Comment:
Could you add comment to explain why we need to reset select list here?
And SelectStmt is one kind of QueryStmt, what about SetOperationStmt?

Reply:
And release rebase the master to pass the UT

Reply:
Done, SetOperationStmt need to handle too